sandstone/filewatcher.py

```diff
@@ -0,0 +1,43 @@
+from watchdog.observers import Observer
+import watchdog
+
+class Filewatcher(watchdog.events.FileSystemEventHandler):
+    """
+    Starts a watchdog instance to watch over the filesystem
+    Handles events - CREATE, REMOVE,MOVE
+    """
+    watched_directories = []
+
+    def __init__(self):
+        print 'Watchdog instance started'
+
+    def add_directory_to_watch(self, directory):
```

Comment:
Since `watched_directories` is a static member of this class, and the `observer` does not get bound to an instance, this should be a `classmethod`.